index.html

```diff
@@ -10,8 +10,190 @@
       rel="stylesheet"
     />
-    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
-    <title>Creative Commons Global Summit 2015</title>
+
+    <link rel="stylesheet" href="style.css" />
+    <title>Europe Global Summit 2023</title>
   </head>
-  <body></body>
+  <body>
+    <section class="main">
+      <div class="mobile-menu">
+        <img
+          src="./images/ClosingIcon.png"
+          alt="closing icon"
+          class="closing-icon"
+        />
+        <a href="index.html" class="logo-imgs"
+          ><img src="images/earth-globe.png" class="mobile-logo" alt="logo"
+        /></a>
+        <ul class="navigation">
+          <li class="menu"><a href="about.html" class="hover">About</a></li>
+          <li class="menu"><a href="#program" class="hover">Program</a></li>
+          <li class="menu"><a href="#join" class="hover">Join</a></li>
+          <li class="menu"><a href="#sponsor" class="hover">Sponsor</a></li>
+          <li class="menu"><a href="#news" class="hover">News</a></li>
+        </ul>
+      </div>
+      <div class="nav-left">
+        <img src="images/enabled.jpg" alt="Enable icon" class="enabled-menu" />
+      </div>
+      <header class="desktop-menu">
+        <div class="pre-header">
+          <div class="pre-header-flex">
+            <ion-icon name="logo-facebook" class="icon"></ion-icon>
+            <ion-icon name="logo-twitter" class="icon"></ion-icon>
+            <span class="tabs">English</span>
+            <span class="tabs">Language</span>
+            <span class="tabs">Logout</span>
+          </div>
+        </div>
+        <div class="header">
+          <a href="index.html" class="logo-imgs"
+            ><img src="images/earth-globe.png" class="header-logo" alt="logo"
+          /></a>
+          <ul class="uo-list-header">
+            <li class="menu"><a href="about.html" class="hover">About</a></li>
+            <li class="menu"><a href="#program" class="hover">Program</a></li>
+            <li class="menu"><a href="#join" class="hover">Join</a></li>
+            <li class="menu"><a href="#sponsor" class="hover">Sponsor</a></li>
+            <li class="menu"><a href="#news" class="hover">News</a></li>
+          </ul>
+          <a class="button-nav">Europe Global Summit</a>
+        </div>
+      </header>
+      <div class="headings">
+        <p class="sub-heading">"Hello! Europe"</p>
+        <h1 class="main-text">Europe Global Summit 2023</h1>
+      </div>
+      <div class="border-text">
+        <p class="border-paragraph">
+          Europe Summit is the biggest gathering events in the Europe. It unites
+          all of the countries of Europe with providing speakers and
+          presentations from the rest of the World.
+        </p>
+      </div>
+      <div class="date">
+        <h2 class="date-heading">06.02.2023(MON) - 18(SAT)</h2>
+        <span class="date-span"
+          >@ Geneva, Switzerland, Geneva Center for the Arts</span
```

Comment:
- [ ] While you did a good job styling your project :1st_place_medal:, It is required to use the same background from the original design. [Here](https://ingdavidtm.github.io/capstone-biotechnology/images/backgroundHeader.png) you can find the required background.

**Your design**

![Screenshot from 2023-02-07 07-54-02](https://user-images.githubusercontent.com/105117832/217252208-8e49a526-26fc-4b55-8530-2ce2a1b6f508.png)


**Template**

![Screenshot from 2023-02-07 04-46-32](https://user-images.githubusercontent.com/105117832/217210339-690f253b-f4a6-418d-81ba-b29dc4ab0b86.png)